src/provider/types.ts

```diff
@@ -12,6 +12,7 @@ interface BaseBitcoinProvider {
   signTransaction: (request: string) => Promise<SignTransactionResponse>;
   sendBtcTransaction: (request: string) => Promise<SendBtcTransactionResponse>;
   createInscription: (request: string) => Promise<CreateInscriptionResponse>;
+  createRepeatInscriptions: (request: string) => Promise<CreateInscriptionResponse>;
```

Comment:
could we create the duplicate CreateRepeatInscriptionResponse type, in case it changes in the future, it won't require consuming apps to refactor

Reply:
ok I also changed this